backend/python/app/services/implementations/story_service.py

```diff
@@ -55,6 +55,21 @@ def create_story(self, story, content):
 
         return new_story
 
+    def get_stories_available_for_translation(self, user):
```

Comment:
Found an error: 
```
stories 
id | title |       description       |                youtube_link                 | level | translated_languages
----+-------+-------------------------+---------------------------------------------+-------+----------------------
  5 | cill  | heres my storyy         | https://www.youtube.com/watch?v=5qap5aO4i9A |     1 | {ARABIC}
  6 | cill  | hindi story!!!          | https://www.youtube.com/watch?v=5qap5aO4i9A |     1 | {HINDI}
  7 | cill  | french story!!!         | https://www.youtube.com/watch?v=5qap5aO4i9A |     1 | {FRENCH}
  8 | cill  | french + hindi story!!! | https://www.youtube.com/watch?v=5qap5aO4i9A |     1 | {FRENCH,HINDI}
  
  users
  id | first_name  | last_name |           auth_id            | role  | resume | profile_pic |   approved_languages
----+-------------+-----------+------------------------------+-------+--------+-------------+-------------------------
  2 | PlanetAdmin | Read      | rHBoE0GuVFThaWgEUYj89IZ9QvZ2 | Admin |        |             | {"HINDI":3, "FRENCH":3}
```
`get_stories_available_for_translation` returns stories 5, 6, 7.

However, this reminds me that the language filter will always be set on the frontend. Let's have `get_stories_available_for_translation` accept a `language` rather than a `user`. Sorry that I didn't make this realization earlier 🤦

Reply:
You'll actually still need the `user` to an extent - you'll still have to look at the approved_languages on the `user` to make the appropriate level filter.